supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -5,5 +5,117 @@
+
+----
+Before this update,  <X problem> caused <Y situation> (OPTIONAL: under the following <Z conditions>). With this update, <fix> resolves the issue (OPTIONAL: and <agent> can <perform operation> successfully).
+----
+
+Example doc text:
+----
+“Before this update, the loopback device for Cinder iSCSI/LVM backend was not recreated after a system restart, which prevented the cinder-volume service from restarting. With this update, a systemd service recreates the loopback device and the Cinder iSCSI/LVM backend persists after a restart.”
+----
+
+.Known Issues
+
+Use present tense for the issue and the imperative form for the workaround in the following format:
+
+----
+There is currently a known issue <present tense explanation> under <X conditions>. 
+
+Workaround: <workaround>.
+----
+
+Example doc text:
+----
+"Currently, you cannot use Orchestration (heat) templates with the director to deploy an overcloud that requires NFS as an Image service (glance) back end. There is currently no workaround for this issue."
+----
+
+----
+"The Compute services (nova) might fail to deploy because the nova_wait_for_compute_service script is unable to query the Nova API. If a remote container image registry is used outside of the undercloud, the Nova API service might not finish deploying in time. 
+Workaround: Rerun the deployment command, or use a local container image registry on the undercloud."
+----
+
+.Technology Preview
+
+Use present tense in the following format:
+----
+In <release>, a technology preview is available for <feature explanation>.
+----
+
+Example doc text: 
+----
+"In Red Hat OpenStack Platform 16.0, a technology preview is available for the Bare Metal Provisioning service (ironic) deployed on an IPv6 provisioning network for BMaaS (Bare Metal as-a-Service) tenants."
+----
+
+.Deprecated Functionality
+Warn users about the following deprecation stages:
+
+* Plan to deprecate
+* Deprecate
+* Plan to remove
+* Remove 
+
+If available, inform users of alternative capabilities and workarounds. Use the following format:
+
+Deprecation notice::
+----
+In <product> <release>, <name of capability> is deprecated. Bug fixes and support will be provided through the end of the <releasename or, if unknown, “a future”> lifecycle but no new feature enhancements will be made. (OPTIONAL: You can use <alternative capability> instead.) 
+----
+
+Removal notice::
+----
+In <product> <release>, <name of capability> is removed. Bug fixes and support will be provided through the end of the <releasename or, if unknown, “a future”> life cycle but no new feature enhancements will be made. (OPTIONAL: You can use <alternative capability> instead.) 
+----
+
+Example doc texts:
+----
+"In Red Hat Openstack Platform (RHOSP) 14, the director graphical user interface will be deprecated. Bug fixes and support will be provided through the end of a future life cycle but Red Hat will not implement new feature enhancements."
```

Comment:
I'm not sure if these examples using "will be" are appropriate.

If it's for the existing release's release notes, then it should be "is".

If it's for a future release, I think that we need to be careful about stating specifics. Typically I'd think that we'd normally want to avoid predicting what will happen in the future, as plans might change. @IngridT1 do you have any thoughts or know any rules on this?

Reply:
I wince at the sight of 'will' but from my understanding, warning users about future deprecation in release notes is the only real time it _is_ appropriate to use the future tense. Happy to be wrong though if @IngridT1  has any guidance :)

Reply:
Sorry for the late comment, but I would remove "will", because it's understood here.

Reply:
Removed the 'will's. Thanks!